docs/channels.md

```diff
@@ -0,0 +1,768 @@
+      [%read-at =time]
+      [%watch ~]
+      [%unwatch ~]
+  ==
+```
+
+Response (subscriber to client communication) for a channel.
+### posts
+```hoon
++$  posts  ((mop id-post (unit post)) lte)
+```
+
+Posts indexed by time.
+
+[id-post](#id-post) \| [note](#note)
+### post
+```hoon
++$  post   [reply-seal essay]
+```
+
+An individual post.
+
+[reply-seal](#reply-seal) \| [essay](#essay)
+### reply-seal
+```hoon
++$  reply-seal
+  $:  id=id-post
+      =reacts
+      =replies
+      =reply-meta
+  ==
++$  reacts  (map ship react)
++$  reply   [reply-seal memo]
++$  replies  ((mop id-reply reply) lte)
++$  reply-seal   [id=id-reply =reacts]
+```
+
+Metadata for a post.
+
+[id-reply](#id-reply) \| [memo](#memo) \| [reply-meta](#reply-meta)
+### reply-meta
+```hoon
++$  reply-meta
+  $:  reply-count=@ud
+      last-repliers=(set ship)
+      last-reply=(unit time)
+  ==
+```
+
+Metadata for replies.
+### create-channel
+```hoon
++$  create-channel
+  $:  =kind
+      name=term
+      group=flag:g
+      title=cord
+      description=cord
+      readers=(set sect:g)
+      writers=(set sect:g)
+  ==
+```
+
+Action for creating channels. 
+
+[kind](#kind) | [flag:groups](#flaggroups) | [sect:groups](#sectgroups)
+### sect:groups
+```hoon
++$  sect  term
+```
+
+Groups role.
+### flag:groups
+```hoon
++$  flag  (pair ship term)
+```
+
+Unique identifier for a group.
+### r-channels
+```hoon
++$  r-channels  (map nest r-channel)
+```
+Response (subscriber to client communication) for channels.
+
+### r-channel
+```hoon
++$  r-channel
+  $%  [%posts =kind-data-posts]
+      [%post id=id-post =r-post]
+      [%set-order order=pin-channel-posts]
+      [%view =view]
+      [%sort =sort]
+      [%perm =perm]
+    ::
+      [%create =perm]
+      [%join group=flag:g]
+      [%leave ~]
+    ::
+      [%read ~]
+      [%read-at =time]
+      [%watch ~]
+      [%unwatch ~]
+  ==
+```
+
+Response (subscriber to client communication) for a channel.
+
+### channels
+```hoon
++$  channels  (map nest channel)
+```
+
+A map of [nest](#nest) to [channel](#channel)
+
+### channel
+```hoon
+++  channel
+  |^  ,[global local]
+  +$  global
+    $:  posts=posts
+        order=arranged-posts
+        =view
+        =sort
+        =perm
+    ==
+  ::
+  +$  local
+    $:  =net
+        =remark
+    ==
+  --
+```
+
+Posts and view/sort/order info for a channel
+
+[posts](#posts) \| [arranged-posts](#arranged-posts) \| [view](#view) \| [sort](#sort) \| [perm](#perm)
+
+### perm
+```hoon
++$  perm
+  $:  writers=(set sect:g)
+      group=flag:g
+  ==
+```
+
+Permissions for a channel
+
+[sect:groups](#sectgroups) \| [flag:groups](#flaggroups)
+### v-channel
+Internal representation of a channel. Contains syncing information and revision numbers.
+
+[channel](#channel)
+
+### nest
+```hoon
++$  nest  [=kind =ship name=term]
+```
+
+Uniquely identifies a channel.
+
+[kind](#kind)
+
+### kind
+```hoon
++$  kind  ?(%notebook %gallery %chat)
+```
+
+Channel type.
```

Comment:
I would probably put these at the very top of the types section, and generally order them so that primitives/foundationals come first. Right now the first type is described as "A nest and a note", but we haven't learned what either of those are.